src/python/k4a/src/k4a/_bindings/k4a.py

```diff
@@ -0,0 +1,525 @@
+k4a_image_set_system_timestamp_nsec = _k4a_lib.k4a_image_set_system_timestamp_nsec
+k4a_image_set_system_timestamp_nsec.restype = None
+k4a_image_set_system_timestamp_nsec.argtypes=(_ImageHandle, _ctypes.c_uint64)
+
+
+#K4A_EXPORT void k4a_image_set_exposure_usec(k4a_image_t image_handle, uint64_t exposure_usec);
+k4a_image_set_exposure_usec = _k4a_lib.k4a_image_set_exposure_usec
+k4a_image_set_exposure_usec.restype = None
+k4a_image_set_exposure_usec.argtypes=(_ImageHandle, _ctypes.c_uint64)
+
+
+#K4A_EXPORT void k4a_image_set_white_balance(k4a_image_t image_handle, uint32_t white_balance);
+k4a_image_set_white_balance = _k4a_lib.k4a_image_set_white_balance
+k4a_image_set_white_balance.restype = None
+k4a_image_set_white_balance.argtypes=(_ImageHandle, _ctypes.c_uint32)
+
+
+#K4A_EXPORT void k4a_image_set_iso_speed(k4a_image_t image_handle, uint32_t iso_speed);
+k4a_image_set_iso_speed = _k4a_lib.k4a_image_set_iso_speed
+k4a_image_set_iso_speed.restype = None
+k4a_image_set_iso_speed.argtypes=(_ImageHandle, _ctypes.c_uint32)
+
+
+#K4A_EXPORT void k4a_image_reference(k4a_image_t image_handle);
+k4a_image_reference = _k4a_lib.k4a_image_reference
+k4a_image_reference.restype = None
+k4a_image_reference.argtypes=(_ImageHandle,)
+
+
+#K4A_EXPORT void k4a_image_release(k4a_image_t image_handle);
+k4a_image_release = _k4a_lib.k4a_image_release
+k4a_image_release.restype = None
+k4a_image_release.argtypes=(_ImageHandle,)
+
+
+#K4A_EXPORT k4a_status_t k4a_device_start_cameras(k4a_device_t device_handle, const k4a_device_configuration_t *config);
+k4a_device_start_cameras = _k4a_lib.k4a_device_start_cameras
+k4a_device_start_cameras.restype = EWaitStatus
+k4a_device_start_cameras.argtypes = (_DeviceHandle, _ctypes.POINTER(DeviceConfiguration))
+
+
+#K4A_EXPORT void k4a_device_stop_cameras(k4a_device_t device_handle);
+k4a_device_stop_cameras = _k4a_lib.k4a_device_stop_cameras
+k4a_device_stop_cameras.restype = None
+k4a_device_stop_cameras.argtypes=(_DeviceHandle,)
+
+
+#K4A_EXPORT k4a_status_t k4a_device_start_imu(k4a_device_t device_handle);
+k4a_device_start_imu = _k4a_lib.k4a_device_start_imu
+k4a_device_start_imu.restype = EStatus
+k4a_device_start_imu.argtypes = (_DeviceHandle,)
+
+
+#K4A_EXPORT void k4a_device_stop_imu(k4a_device_t device_handle);
+k4a_device_stop_imu = _k4a_lib.k4a_device_stop_imu
+k4a_device_stop_imu.restype = None
+k4a_device_stop_imu.argtypes = (_DeviceHandle,)
+
+
+#K4A_EXPORT k4a_buffer_result_t k4a_device_get_serialnum(k4a_device_t device_handle,
+#                                                        char *serial_number,
+#                                                        size_t *serial_number_size);
+k4a_device_get_serialnum = _k4a_lib.k4a_device_get_serialnum
+k4a_device_get_serialnum.restype = EBufferStatus
+k4a_device_get_serialnum.argtypes = (_DeviceHandle, 
+    _ctypes.POINTER(_ctypes.c_char), _ctypes.POINTER(_ctypes.c_ulonglong))
```

Comment:
there is a ctpyes c_size_t
That might be more extensible if we want to support both 32 and 64 bit

Reply:
Changed to c_size_t.